src/java.base/share/classes/jdk/internal/reflect/MethodHandleAccessorFactory.java

```diff
@@ -414,4 +456,5 @@ static class LazyStaticHolder {
     }
 
     private static final Unsafe UNSAFE = Unsafe.getUnsafe();
+    private static final ReflectionFactory reflectionFactory = ReflectionFactory.getReflectionFactory();
```

Comment:
As this now accesses shared parameter types, we can change this file to avoid cloning the array of parameter types in `slotCount` and creating a method type from a member.  This can be done in a separate PR.